projects/kit/components/rating/rating.component.ts

```diff
@@ -0,0 +1,105 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    forwardRef,
+    Inject,
+    Input,
+    Optional,
+    Self,
+    ViewChild,
+} from '@angular/core';
+import {NgControl} from '@angular/forms';
+import {
+    AbstractTuiControl,
+    isNativeFocused,
+    TUI_FOCUSABLE_ITEM_ACCESSOR,
+    tuiDefaultProp,
+} from '@taiga-ui/cdk';
+
+import {TUI_RATING_OPTIONS, TuiRatingOptions} from './rating-options';
+
+@Component({
+    selector: 'tui-rating',
+    templateUrl: './rating.template.html',
+    styleUrls: ['./rating.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: TUI_FOCUSABLE_ITEM_ACCESSOR,
+            useExisting: forwardRef(() => TuiRatingComponent),
+        },
+    ],
+})
+export class TuiRatingComponent extends AbstractTuiControl<number> {
```

Comment:
Please add that it implements focusable accessor interface

Reply:
fixed